### ISSUE TITLE
How activation process is generated?

### ISSUE THREAD

**Issue by vladmocan2000** [johnolafenwa/DeepStack#110]
@johnolafenwa 
**environment**
Raspberry Pi
Raspbian OS
NCS 2
Home Assistant framework

**What I'm interested for?**
As far as I understand activation on Raspberry Pi is not supported anymore, however I am interested about activation process.
I was searching along the code and I wasn't able to identify the piece of code responsible for this action.
My questions are: 
1. Where I can find the code?
2. Based on my experience following the activation process, when the activation is successfully, a file must be generated and locally stored. Where I can find this file? Or maybe the process is different.

The last question is in which manner I can use NCS 2 and when this feature will be supported in the future?

**Comment by CarpeDiemRo** [comment]
The subject preoccupied me, as well. I'm using **NCS2** with **HA** and when I will reinstall the system I won't be able to activate Deepstack. 

I am wondering if a workaround is possible for all those who use Deepstack in conjunction with RPI and NCS2?

**Comment by OlafenwaMoses** [comment]
@CarpeDiemRo @vladmocan2000 We have released a version for ARM devices which include Raspberry Pi and other edge chips and doesn't require an activation key. Follow the links below.

https://forum.deepstack.cc/t/announcing-deepstack-for-raspberry-pi-and-all-arm64-devices-new-deepstack-cpu-gpu-and-jetson-release/986

https://docs.deepstack.cc/arm64/

**Comment by CarpeDiemRo** [comment]
@OlafenwaMoses Thank you for the answer. I have already installed the suggested version, but as far as I saw it is a "CPU processing" rather than NCS processing. It is correct or, maybe, I missed something.

My intention is to use in the future NCS2 stick that worked perfectly on my environment.

**Comment by OlafenwaMoses** [comment]
@CarpeDiemRo That's correct. We dropped support for NCS because it gave us compatibility issues with new functionalities we are adding and with the unified intelligence stack that works across all DeepStack's versions. We want to ensure every version of DeepStack all support the same APIs and the NCS support was a drawback in this direction.

Running the ARM version on a Raspberry Pi with higher memory (8GB Ram RaspberryPi 4) will offer much faster API speed + you can also work with the NVIDIA Jetson devices as well; they provide much faster inference API speed.

**Comment by CarpeDiemRo** [comment]
@OlafenwaMoses Got it. However, I have more questions. 

1. As I mentioned I'm using RPI in conjunction with NCS2. 
Question1: What will happen when the License will expire? I will still be able to use Deepstack? 
![Screenshot 2021-08-24 at 14 58 51](https://user-images.githubusercontent.com/38489167/130612765-1df43e9b-1b25-402d-92d4-e37c5ac71cc2.png)
2. I was try hard to use Deepstack in the new CPU RPI version, but, even though I'm using RPI 8GB the hardware is not able to run smoothly HomeAssistant and Deepstack / 4 cameras / on the same device. 
Question2: Do you plan to develop in the near future Deepstack with NCS2? I am sure that a lot of Home Assistant enthusiasts already use this solution and will be happy to maintain it. 
3. I saw that the licensing is not available anymore. 
Question3: Could you please mention where the licensing method is placed in the code in order to try to avoid the licensing. Doing at my own risk, of course.
